nilearn/image/image.py

```diff
@@ -172,8 +172,8 @@ def _smooth_array(arr, affine, fwhm=None, ensure_finite=True, copy=True):
         filtering.
 
     copy: bool
-        if True, input array is not modified. False by default: the filtering
-        is performed in-place.
+        if True, input array is not modified. True by default: the filtering
+        is not performed in-place.
```

Comment:
Given that it's an internal function, I would leave it not doing copies.

Reply:
The default in `_crop_img_to` is `copy=True`, but the documentation is wrong. I just fixed that as I copied the text over to `threshold_img`. Did I understand correctly that you'd like to change that to `copy=False`?